style.css

```diff
@@ -0,0 +1,178 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=Lexend+Deca&display=swap');
+
+:root {
+  --grey: #979797;
+  --dark: #1B1937;
+  --white: #FFFFFF;
+  --purple: #AB5CDB;
+}
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: 'Inter', sans-serif;
+}
+body {
+  background-color: black;
+}
+
+main, footer {
+  width: 327px;
+  display: flex;
+  margin: 0 auto;
+  margin-top: 30px;
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
+}
+
+h1, dd {
+  line-height: auto;
+  text-align: center;
+  color: var(--white);
+  letter-spacing: 0.5px;
+}
+
+p, dt {
+  color: var(--grey);
+  text-align: center;
+  letter-spacing: 0.5px;
+}
+
+.img {
+  overflow: hidden;
+  margin-bottom: -4px;
+  border-radius: 8px 8px 0 0;
+  background-color: var(--purple);
+}
+
+img {
+  opacity: 0.4;
+  height: 240px;
+  object-fit: cover;
+}
+
+.info-wrapper {
+  padding: 20px;
+  border-radius: 0 0 8px 8px;
+  background-color: var(--dark);
+}
+
+h1 {
+  font-size: 28px;
```

Comment:
Static font sizes are not responsive to the user preference. Use relative units like rem or em instead.